libs/lib-client/installer/src/lib/installer.service.ts

```diff
@@ -0,0 +1,11 @@
+import { EventEmitter, Injectable } from '@angular/core';
+import { ActivitySchema } from '@flogo-web/core';
+
+@Injectable()
+export class InstallerService {
```

Comment:
Can we rename this to `ContribInstallerService`? Since this is intended to be consumed in other modules I think being more explicit about the intent of the service will be beneficial for readability.

Reply:
Done. Renamed.